src/limbo/opt/cmaes.hpp

```diff
@@ -41,12 +43,12 @@ namespace limbo {
                 int irun, lambda = 0, countevals = 0;
                 char const* stop;
                 boundary_transformation_t boundaries;
-                double lowerBounds[] = {0.0}; // TODO  put this into params?
+                double lowerBounds[] = {0.0}; 
                 double upperBounds[] = {1.006309}; // Allows solution to be pretty close to 1
```

Comment:
Like the comment says, with it the solutions get really close to 1. If we put 1, the maximum that can be reached is a little lower than 1.